gridstatus/ieso.py

```diff
@@ -1,7 +1,9 @@
 import time
 import xml.etree.ElementTree as ET
+from io import BytesIO
 
 import pandas as pd
+import pycurl
```

Comment:
I don't know why PycURL is able to access the `.ashx` URL with SSL verification while requests cannot. I tried:

* `requests` with/without [`certifi`](https://pypi.org/project/certifi/) 
* `urllib3` with/without `certifi`
* `http.client` with/without `certifi`

Something about this specific URL causes issues with `requests`. I noticed the URL works fine with cURL (which does SSL verification by default) so found a library that uses cURL.